src/lib/middleware/hardcodedRedirects.ts

```diff
@@ -82,6 +82,7 @@ const redirects = {
   "/user/purchases": "/settings/purchases",
   "/user/payments": "/settings/payments",
   "/user/alerts": "/settings/alerts",
+  "/page/collector-faqs-selling-on-artsy": "/consign",
```

Comment:
How did you know to do this?

Reply:
https://github.com/artsy/force/blob/3ebd208040dc8d89da0acd6add8aacc6e3320ae8/src/desktop/apps/page/routes.coffee#L11-L12